plugins/auth-backend-module-aws-alb-provider/src/helpers.ts

```diff
@@ -70,6 +70,17 @@ export const makeProfileInfo = (
   };
 };
 
+const getPublicKeyEndpoint = (region: string) => {
+  const commercialEndpoint = `https://public-keys.auth.elb.${encodeURIComponent(
+    region,
+  )}.amazonaws.com/`;
+  const govEndpoint = `https://s3-${encodeURIComponent(
+    region,
+  )}.amazonaws.com/aws-elb-public-keys-prod-${encodeURIComponent(region)}/`;
+
+  return region.startsWith('us-gov') ? govEndpoint : commercialEndpoint;
+};
```

Comment:
looking at the [AWS Docs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html) it seems like there are only 2 "us-gov" regions. Happy to keep it like this and generalise more in case there are non-us GovCloud regions in the future.